app.rb

```diff
@@ -0,0 +1,149 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+require 'pry'
+class App
+  def run
+    @books = []
+    @person = []
+    @rental = []
+    puts 'Welcome to School Library App!'
+    puts "\n"
+    options
+  end
+
+  def options
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all retnals for a given person id'
+    puts '7 - Exit'
+    num = gets.chomp.to_i
+    perform_switch(num)
+    options
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def perform_switch(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
```

Comment:
- [ ] Let's work on fixing this issue in the image below. When I create a student and receive a successful message, then I try to list the created `list_people` I find that a person is not properly created.

![student](https://user-images.githubusercontent.com/53124548/158023422-5a0c2cbb-6f18-46a1-81b8-3d6141c19a0b.png)